artifact/gem5art/artifact/artifact.py

```diff
@@ -275,6 +322,8 @@ def _getSerializable(self) -> Dict[str, Union[str, UUID]]:
         data['inputs'] = [input._id for input in self.inputs]
         data['cwd'] = str(data['cwd'])
         data['path'] = str(data['path'])
+        data['supported_gem5_versions'] = str(self.supported_gem5_versions)
+        data['extra'] = str(self.extra)
```

Comment:
Will `str` always work here? should we serialize extra? (I.e.,`json.dumps(extra)`)?

Reply:
fixed